client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -298,7 +312,11 @@ const Checkout = createReactClass( {
 		}
 
 		if ( ! selectedSiteSlug ) {
-			return '/checkout/thank-you/features';
+			return '/checkout/thank-you/features/';
+		}
+
+		if ( isChecklistTest && cartItems.hasPlan( cart ) && this.isNewSite() ) {
```

Comment:
If I'm reading this correctly, this is where we check for a paid plan. This would be a good spot to change the a/b test name so that it only shows for Paid users.